website/docs/installation.md

```diff
@@ -62,6 +62,17 @@ line:
 brew install --cask neovide
 ```
 
+Neovide register launched shells taking the users preferred shell into account.
```

Comment:
I think this should read "Neovide registers launched shells taking the user's preferred shell into account."

But that said, I'm not sure I understand what this is trying to tell the end user.

Reply:
thank you!